packages/babel-plugin-minify-simplify/src/index.js

```diff
@@ -780,6 +780,17 @@ module.exports = ({ types: t }) => {
         exit(path) {
           const { node, parent } = path;
 
+          if (
+            t.isArrowFunctionExpression(parent) &&
+            node.body &&
```

Comment:
We are in a `BlockStatement` - is this check every `falsey` ? A BlockStatement's body is always an array.